scripts/publish-npm.js

```diff
@@ -66,73 +65,17 @@ const argv = yargs
     type: 'boolean',
     default: false,
   })
-  .option('h', {
-    alias: 'include-hermes',
-    type: 'boolean',
-    default: false,
-  }).argv;
+  .strict().argv;
 const nightlyBuild = argv.nightly;
 const dryRunBuild = argv.dryRun;
-const includeHermes = argv.includeHermes;
 const isCommitly = nightlyBuild || dryRunBuild;
 
-saveFilesToRestore(tmpPublishingFolder);
-
-if (includeHermes) {
-  const HERMES_INSTALL_LOCATION = 'sdks';
-  const HERMES_SOURCE_DEST_PATH = `${HERMES_INSTALL_LOCATION}/hermes`;
-
-  let hermesReleaseTag;
-  let hermesReleaseURI;
-  if (isCommitly) {
-    // use latest commit / tarball
-    hermesReleaseURI = 'https://github.com/facebook/hermes/tarball/main';
-  } else {
-    // use one configured in disk
-    fs.readFile(
-      `${HERMES_INSTALL_LOCATION}/.hermesversion`,
-      {
-        encoding: 'utf8',
-        flag: 'r',
-      },
-      function (err, data) {
-        if (err) {
-          echo('Failed to read current Hermes release tag.');
-          // TODO: We'll need to make sure every release going forward has one of these.
-          exit(1);
-        } else {
-          hermesReleaseTag = data.trim();
-          hermesReleaseURI = `https://github.com/facebook/hermes/archive/refs/tags/${hermesReleaseTag}.tar.gz`;
-        }
-      },
-    );
-  }
-
-  const tmpDownloadDir = fs.mkdtempSync(
-    path.join(os.tmpdir(), 'hermes-tarball'),
-  );
-  const tmpExtractDir = fs.mkdtempSync(path.join(os.tmpdir(), 'hermes'));
-
-  const hermesInstallScript = `
-    mkdir -p ${HERMES_SOURCE_DEST_PATH} && \
-    wget ${hermesReleaseURI} -O ${tmpDownloadDir}/hermes.tar.gz && \
-    tar -xzf ${tmpDownloadDir}/hermes.tar.gz -C ${tmpExtractDir} && \
-    HERMES_SOURCE_EXTRACT_PATH=$(ls -d ${tmpExtractDir}/*) && \
-    mv $HERMES_SOURCE_EXTRACT_PATH ${HERMES_SOURCE_DEST_PATH}
-  `;
-
-  if (fs.existsSync(`${HERMES_SOURCE_DEST_PATH}`)) {
-    if (exec(`rm -rf ./${HERMES_SOURCE_DEST_PATH}`).code) {
-      echo('Failed to clean up previous Hermes installation.');
-      exit(1);
-    }
-  }
-  if (exec(hermesInstallScript).code) {
-    echo('Failed to include Hermes in release.');
-    exit(1);
-  }
+if (!argv.help) {
+  echo(`The temp publishing folder is ${tmpPublishingFolder}`);
 }
 
+saveFilesToRestore(tmpPublishingFolder);
```

Comment:
headsup - I think this method `saveFilesToRestore` can be completely removed - it was saving some stuff and then was getting re-added, but that "re-introduction" part was removed by this PR: https://github.com/facebook/react-native/pull/34846/files

so basically (as far as I can tell) this method is not worth invoking because we are not restoring them anyway? 🤷‍♂️ so we could probably wipe out entirely